RosettaCore/src/main/java/com/hubspot/rosetta/Rosetta.java

```diff
@@ -17,8 +17,8 @@
 public enum Rosetta {
   INSTANCE;
 
-  private final List<Module> modules = new CopyOnWriteArrayList<Module>(defaultModules());
-  private final AtomicReference<ObjectMapper> mapper = new AtomicReference<ObjectMapper>(cloneAndCustomize(new ObjectMapper()));
+  private static final List<Module> MODULES = new CopyOnWriteArrayList<Module>(defaultModules());
+  private static final AtomicReference<ObjectMapper> MAPPER = new AtomicReference<ObjectMapper>(cloneAndCustomize(new ObjectMapper()));
```

Comment:
That seemed to cause a weird error in class initialization after I changed `cloneAndCustomize` to a static method. I'm really only using enum to enforce singleton and non-instantiability